CocoaLumberjack.podspec

```diff
@@ -32,6 +32,12 @@ Pod::Spec.new do |s|
 
   s.subspec 'Swift' do |ss|
     ss.dependency 'CocoaLumberjack/Core'
-    ss.source_files = 'Sources/CocoaLumberjackSwift/**/*.swift', 'Sources/CocoaLumberjackSwiftSupport/include/**/*.{h}'
+    ss.source_files = 'Sources/CocoaLumberjackSwift/**/*.swift',  'Sources/CocoaLumberjackSwiftLogBackend/**/*.swift', 'Sources/CocoaLumberjackSwiftSupport/include/**/*.{h}'
+  end
+
+  s.subspec 'SwiftLog' do |ss|
+    ss.dependency 'SwiftLogAPI'
```

Comment:
This needs to have some kind of version. We're not compatible with all versions of SwiftLog. See my comment in #1354

Reply:
Okay, what version should I specify? If I understand it correctly, v1.5.0 added support for metadata providers.

Since they seem to use semantic versioning, I'll pin it to >= 1.5 and < 2.0.

Reply:
Yea, 1.5.0..<2.0.0 would be what the Package.swift currently specifies.

But I'm already working on automating the CocoaPods handling. As mentioned in #1354 I'd rather reduce the maintenance burden than to increase it. So I'll probably close this PR later and use the generated version 😉